certified-connectors/Sociabble/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1073 @@
+        "operationId": "GetAudiences",
+        "parameters": [],
+        "x-ms-visibility": "internal",
+        "description": "Get audiences"
+      }
+    },
+    "/channels": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "An array of channels",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "id": {
+                    "type": "string",
+                    "description": "id"
+                  },
+                  "name": {
+                    "type": "string",
+                    "description": "name"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get channels",
+        "operationId": "GetChannels",
+        "x-ms-visibility": "internal",
+        "parameters": [],
+        "description": "Get channels"
+      }
+    },
+    "/content/internalnews": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "An internal news",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "id": {
+                  "type": "string",
+                  "description": "The unique identifier of the content",
+                  "title": "Id"
+                },
+                "contents": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "language": {
+                        "type": "string",
+                        "description": "Language",
+                        "title": "language"
+                      },
+                      "isDefault": {
+                        "type": "boolean",
+                        "description": "Is the language default",
+                        "title": "language default ?"
+                      },
+                      "title": {
+                        "type": "string",
+                        "description": "Title",
+                        "title": "title"
+                      },
+                      "summary": {
+                        "type": "string",
+                        "description": "A short summary of the content",
+                        "title": "summary"
+                      },
+                      "description": {
+                        "type": "string",
+                        "description": "Content",
+                        "title": ""
+                      },
+                      "mediaIds": {
+                        "type": "array",
+                        "items": {
+                          "type": "string"
+                        },
+                        "description": "mediaIds"
+                      }
+                    }
+                  },
+                  "description": "",
```

Comment:
@mdujardibs Please provide valid description

Documentation -[Define operation and parameter summaries and descriptions](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

Reply:
Modifications done.

Reply:
@mdujardibs Can you please push your changes to the PR, the description values are not fixed

Reply:
@sriyen-msft I actually did push a commit the 16th of August, https://github.com/microsoft/PowerPlatformConnectors/pull/1026/commits/646008f202b088aaf9b544b9b61602bae34c28c7

Sorry, I am not use  to github, do I need to do something else ?